js/google_ima.js

```diff
@@ -987,6 +1000,7 @@ require("../html5-common/js/utils/utils.js");
         if (_IMAAdsManager)
         {
           this.currentIMAAd = null;
```

Comment:
These two lines with this.variable can live outside of the if check.  That way we insure they'll get cleared.